### PR TITLE
fix blue download btn

### PR DESCRIPTION
**Pull request by nicolasboeuf**
Fixing [ticket #965](https://github.com/etalab/data.gouv.fr/issues/965)
Working in Safari 15.2 + android browsers
To check on iOS device

### CODE REVIEW

**Review by maudetes — COMMENTED** [review]
I could not reproduce the bug since I don't have Safari, thus can't really check the fix.
I'm assigning @nicolaskempf57 that will know better for this PR :wink: :pray:

File: src/components/Table.vue [hunk 1]
```diff
@@ -663,6 +663,7 @@ tfoot .fr-btn--secondary {
   --border-active-blue-france: var(--text-inverted-grey);
   --text-action-high-blue-france: var(--text-inverted-greys);
```
Comment (nicolaskempf57) [inline src/components/Table.vue:664]:
```suggestion
  --text-action-high-blue-france: var(--text-inverted-grey);
```

**Review by nicolaskempf57 — COMMENTED** [review]
There was an error in the variable name. This should be enough without adding another `color`.

**Comment by nicolasboeuf** [comment]
Yep, well spotted @nicolaskempf57. Change commited.

**Review by nicolaskempf57 — APPROVED** [review]

**Pull request merged**